roles/openshift_monitor_availability/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+openshift_monitor_app_create_image: quay.io/redhat/openshift-monitor-project-lifecycle:0.0.1
```

Comment:
This should look something like this:
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_descheduler/defaults/main.yaml#L8-L17

Don't necessarily need to specify a unique registry for origin and openshift-enterprise, I more care about the version.  We need to at least track "openshift_image_tag" for openshift-enterprise if we can't use openshift_image_tag for both origin and enterprise.

Reply:
I refactored to be more like the bit you reference. I'm making use of `openshift_deployment_type`, but I'm not doing anything with `openshift_image_tag` since I don't understand what relationship it has to the image I'm working with.